pepys_import/core/store/data_store.py

```diff
@@ -44,6 +45,23 @@
 DEFAULT_DATA_PATH = os.path.join(PEPYS_IMPORT_DIRECTORY, "database", "default_data")
 USER = getuser()  # Login name of the current user
 
+# Constant Lists of revisions - list all files in the directory
+SQLITE_REVISIONS_FOLDER = os.path.join(MIGRATIONS_DIRECTORY, "sqlite_versions")
+SQLITE_REVISION_LIST = os.listdir(SQLITE_REVISIONS_FOLDER)
+SQLITE_REVISION_IDS = []
+if len(SQLITE_REVISION_LIST) > 0:
+    for sqlite_revision in SQLITE_REVISION_LIST:
```

Comment:
This loop is fine, but could probably be done more 'pythonicly' with a list comprehension. Have a google to learn about them - as you'll find them elsewhere in the codebase - and then have a go at rewriting as a list comprehension